src/amo/components/AddonReviewList/index.js

```diff
@@ -40,11 +44,12 @@ import type { I18nType } from 'core/types/i18n';
 import './styles.scss';
 
 type Props = {|
-  i18n: I18nType,
   addon: AddonType | null,
+  addonIsLoading: boolean,
```

Comment:
Can you create `InternalProps` and move all the mapped and injected props there? I'm not sure why this component never got it.

Reply:
Sure, I'll do that.

Reply:
Hmm, This component doesn't actually have any `Props` that aren't internal. I added a type for `Props` that I defined as `{||}` to use for the component declaration.